### PR TITLE
[16.0][ADD] sale_loyalty_industry

### PR DESCRIPTION
**Pull request by santostelmo**
This module allow to restrict the industries in which the sale loyalty program applies.

### CODE REVIEW

File: sale_loyalty_industry/tests/test_program_rules.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        self.immediate_promotion_program.industry_ids = self.industry_A
+        self.steve.industry_id = self.industry_B
+        order = self._create_sale_order()
+        self._auto_rewards(order, self.immediate_promotion_program)
+        self.assertEqual(
+            len(order.order_line.ids),
+            2,
+            "The promo offer is not applied to industry_B.",
+        )
+        self.immediate_promotion_program.industry_ids = self.industry_B
```
Comment (simahawk) [inline sale_loyalty_industry/tests/test_program_rules.py:69]: these are all different cases, right? If yes, rule of thumb: define 1 method for each of them unless it gets too complicated w/ the setup

**Review by simahawk — APPROVED** [review]
LG overall, minor remark for tests

File: sale_loyalty_industry/tests/test_program_rules.py (hunk 1 above)
Comment (santostelmo) [reply sale_loyalty_industry/tests/test_program_rules.py:69]: Done splitted unit test in several ones

**Comment by santostelmo** [comment]
Moved that PR to sale-promotion: https://github.com/OCA/sale-promotion/pull/197

**Pull request closed without merging**